wolfssl/wolfcrypt/settings.h

```diff
@@ -314,8 +314,26 @@
         #define WOLFSSL_USER_SETTINGS
     #endif /* WOLFSSL_USER_SETTINGS */
 
-    /* board-specific */
+    /* Board-specific network capability check */
```

Comment:
I still don't understand why this is needed. Why forcefully disable all TLS in the else case? Why not allow it but only with WOLFSSL_USER_IO?

Reply:
Good point. I've removed this section from the PR.